### ISSUE TITLE
Getting Invalid Response Status Error

### ISSUE THREAD

**Issue by Kennyc1012** [bodyscape/cielo_home#20]
Early this morning the integration stopped working and the logs were being flooded with this log:

```
Logger: custom_components.cielo_home.cielohome
Source: custom_components/cielo_home/cielohome.py:259
First occurred: 6:08:13 AM (399 occurrences)
Last logged: 6:23:38 AM
```

```
498, message='Invalid response status', url=URL('wss://apiwss.smartcielo.com/websocket/?sessionId=chrome-1693476893&token=REDACTED
```

Using the Cielo app and google home integration works as expected.

**Comment by SaltyITSailor** [comment]
I am also seeing the “498, message='Invalid response status'” error.

**Comment by bodyscape** [comment]
yup me too, i'm on it! They change something in the auth proces. i don't know what yet

**Comment by JimmyMPage** [comment]
I am seeing this as well. Just to add to the data. Same error in logs.

**Comment by Kuechenfenster** [comment]
the integration of the Breeze Plus devices seems to be offline

**Comment by bodyscape** [comment]
i fixed it, install the version 1.5.3
